src/follower.py

```diff
@@ -186,9 +187,10 @@ def follow(self):
 def main():
     """Instantiates a Follower object and continously calls Follower.follow()."""
     follower = Follower()
+    log = Log()
     while True:
         follower.follow()
-
+        log.log_vehicle_data(follower)
```

Comment:
Why are you still passing the entire follower object into logging?

Reply:
I'm not. I just misunderstood John when he said not to modify the follower file. He just meant dont change opencv_to_wheels so I updated the follower.py